frontend-react/src/components/DataDashboard/ReportDetails/ReportDetails.tsx

```diff
@@ -1,16 +1,49 @@
 import { GridContainer } from "@trussworks/react-uswds";
-import { Helmet } from "react-helmet-async";
 import React from "react";
+import { useParams } from "react-router-dom";
 
 import { FeatureName } from "../../../AppRouter";
+import { AuthElement } from "../../AuthElement";
+import { MemberType } from "../../../hooks/UseOktaMemberships";
+import Crumbs, { CrumbsProps } from "../../Crumbs";
+import { useReportsDetail } from "../../../hooks/network/DataDashboard/DataDashboardHooks";
+import { withCatchAndSuspense } from "../../RSErrorBoundary";
+import HipaaNotice from "../../HipaaNotice";
 
-export function ReportDetails() {
+import { ReportDetailsSummary } from "./ReportDetailsSummary";
+import ReportDetailsTable from "./ReportDetailsTable";
+
+function ReportDetails() {
+    const crumbProps: CrumbsProps = {
+        crumbList: [
+            { label: FeatureName.DATA_DASHBOARD, path: "/data-dashboard" },
+            { label: "Report Details" },
+        ],
+    };
+    const { reportId } = useParams();
+    const { reportDetail } = useReportsDetail(reportId!!);
```

Comment:
is the `!!` to convert a value to a boolean? can we be more verbose, and explicitly declare what we want? maybe we need to change what `useReportsDetail` returns

Reply:
!! after the variable is to tell typescript that you expect reportId to be defined (useParams from react-router types all params as optional which is a pain)

Reply:
I see, wouldn't defining it in an `interface` also suffice?